src/PaymentMethods/iDin/iDin.php

```diff
@@ -26,12 +26,12 @@
 use Buckaroo\PaymentMethods\iDin\Service\ParameterKeys\IssuerAdapter;
 use Buckaroo\PaymentMethods\PaymentMethod;
 
-class IDin extends PaymentMethod
```

Comment:
the class is named correctly `IDin` the imports are not in `/var/www/vendor/buckaroo/sdk/src/PaymentMethods/PaymentMethodFactory.php:167`